DEV-4/XMLParser.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace DEV_4
+{
+  /// <summary>
+  /// This class parses XMLFile.
+  /// </summary>
+  class XMLParser
+  {
+    static void Main(string[] args)
+    {      
+      if (args.Length != 1)
+      {
+        Console.WriteLine("Wrong number of arguments");
+        return;
+      }      
+      string pathXMLFile = args[0];
+      StreamReader file = new System.IO.StreamReader(pathXMLFile);      
+      string xmlLine;
+
+      XMLLineDataExtractor dataExtractor = new XMLLineDataExtractor();
+
+      List<XMLLineData> dataList = new List<XMLLineData>();
+
```

Comment:
use "using" statement when work with resources to be sure they will be correctly disposed if exception case